rhel/SPECS/victoriametrics.spec

```diff
@@ -0,0 +1,59 @@
+%define debug_package %{nil}
+
+%define copying() \
```

Comment:
seems \ is not needed

Reply:
This has been copied from other `*.spec` files. See, for instance, [prometheus.spec](https://github.com/percona/pmm-server-packaging/blob/PMM-2.0/rhel/SPECS/prometheus.spec).